workload/resources/Workload.js

```diff
@@ -677,7 +695,9 @@ class Workload {
       }
       logger.group('Result');
       const results = summarize(inferenceResults.inferenceTimeList);
-      logger.log(`Inference Time: <em style="color:green;font-weight:bolder;">${results.mean.toFixed(2)}+-${results.std.toFixed(2)}</em> [ms]`);
+      const results_median = summarizeMedian(inferenceResults.inferenceTimeList);
```

Comment:
@MarkGHX Please replace the variable name ‘results_median’ with  'resultsMedian' of camelcase style, thanks.

Reply:
@BruceDai Thanks! This is fixed now.